ush/launch_FV3LAM_wflow.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -l
+#!/bin/bash
```

Comment:
@hu5970 We need "-l" on Jet to be able to run the workflow in the crontab

I can bring this back in my PR
 https://github.com/NOAA-GSL/regional_workflow/pull/488